package.json

```diff
@@ -35,6 +35,7 @@
     ]
   },
   "devDependencies": {
-    "@okp4/eslint-config": "^2.0.0"
+    "@okp4/eslint-config": "^2.0.0",
+    "react-router-dom": "^6.16.0"
```

Comment:
react router should be installed as a dependency and not as a devDep.
In fact, this package is used to run your application. 
The packages used in devDep are those that help you during the development phase. 
Here, let's imagine we put your application into production. 
When we will build your app, it will only install your dependencies, in which case react router won't be installed and your application won't have a router :(

Reply:
Thanks for you help !
This issue is resolve in the new commit e759ea72647f7cd249e3e2c28c84bb18a137fe51 👍🏼